stdlib/public/runtime/Errors.cpp

```diff
@@ -312,7 +310,8 @@ reportNow(uint32_t flags, const char *message)
 #define STDERR_FILENO 2
   _write(STDERR_FILENO, message, strlen(message));
 #else
-  write(STDERR_FILENO, message, strlen(message));
+  fputs(message, stderr);
+  fflush(stderr);
```

Comment:
FWIW, `stderr` is normally unbuffered by default, so the `fflush()` is probably not necessary. It won't hurt though.